src/app/components/alter-trip/alter-trip.component.html

```diff
@@ -32,11 +32,11 @@ <h2 class="mb-2 text-lg font-semibold">Itinerary:</h2>
           [inputActivity]="activity"
           (editClicked)="setActivityToEdit(activity)"
           (deleteClicked)="openConfirmModal(activity)" />
-        @if (activityToEdit().id && activityToEdit().id === activity.id) {
+        @if (activityToEdit()?.id && activityToEdit()?.id === activity.id) {
           <div class="flex flex-wrap gap-4">
             <app-itinerary-form
               class="w-full"
-              [activity]="activityToEdit()"
+              [activity]="activityToEdit() ?? undefined"
```

Comment:
If `activityToEdit` can be nullish? Why is the right hand operand set to a nullish value (`undefined`)? My understanding could be off.